upup/pkg/fi/cloudup/openstack/instance.go

```diff
@@ -32,6 +32,9 @@ import (
 const (
 	INSTANCE_GROUP_GENERATION = "ig_generation"
 	CLUSTER_GENERATION        = "cluster_generation"
+	OS_ANNOTATION             = "openstack.kops.io/"
+	BOOT_FROM_VOLUME          = "osVolumeBoot"
```

Comment:
why don't these match the format given above - I would expect `BOOT_FROM_VOLUME = "boot_from_volume"`

Reply:
These values are used in annotations for kops components and they follow the kubernetes standards. Wasn't sure if it is common to have `_` in annotations or use Camel Case. I also didn't see any standard in the metadata for openstack instances.